Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/tools/StampToolIntegrationTest.java

```diff
@@ -203,6 +204,34 @@ public void testCopyPixel() {
 		assertEquals("Second Pixel not Black after using Stamp for copying", Color.BLACK, pixelToControl);
 	}
 
+	@Test
+	public void testStampToolNotCapturingOtherLayers() {
+		PointF surfaceCenterPoint = getScreenPointFromSurfaceCoordinates(perspective.surfaceCenterX, perspective.surfaceCenterY);
+		onView(isRoot()).perform(touchAt(surfaceCenterPoint.x, surfaceCenterPoint.y - Y_CLICK_OFFSET));
```

Comment:
`getScreenPointFromSurfaceCoordinates` is deprecated. i think it should be replaced with something like
```
onDrawingSurfaceView()
	.perform(touchAt(DrawingSurfaceLocationProvider.MIDDLE));
```